osm2streets/src/intersection.rs

```diff
@@ -295,6 +317,39 @@ impl StreetNetwork {
             },
         )
     }
+
+    /// Calculates and fills in the turn movements for an intersection.
+    fn update_movements(&mut self, i: IntersectionID) {
+        let intersection = self.intersections.get_mut(&i).unwrap();
+
+        match intersection.kind {
+            IntersectionKind::Connection => {
+                for turn in intersection.turns.iter_mut() {
```

Comment:
```suggestion
                for turn in &mut intersection.turns {
```

Optional, not really sure what style is preferable. If we later need to do anything else with the iterator, like `.enumerate()`, then how you have it is nicer (because we'd switch back to it)